csrc/ir_interface_nodes.h

```diff
@@ -544,6 +544,9 @@ class TORCH_CUDA_CU_API TensorView : public Val {
   // example, grouping multiple reductions.
   void updateMaxProducerPosition();
 
+  // commit the current changes in leaf domain into rFactor domain
```

Comment:
Can you expand the comment here as the usage of this function needs to be done very carefully. Some more info like intended usages would also be helpful.

Reply:
added